### ISSUE TITLE
Remove dependencies for gnu coreutils

### ISSUE THREAD

**Issue by slomo** [liqd/adhocracy3#1764]
Instead of requiring OSX and BSD users to install core utils we should eliminate the usage of gnu coreutils specific flags from our install scripts.

Particular issues:

* usage of [`cp -s`]( https://github.com/liqd/adhocracy3.mercator/blob/19e78ce0d6868b066c6916be63834410913ad94f/src/adhocracy_frontend/checkcode_and_compile.cfg#L240)

Ideally the POSIX standard should be consulted, I tend to rather check with the freebsd manpages.

**Comment by der-john** [comment]
according to (IEEE)[http://pubs.opengroup.org/onlinepubs/9699919799/utilities/cp.html#tag_20_24_02], I think I can replace `cp -s` by `cp -P`...

**Comment by xi** [comment]
> according to IEEE, I think I can replace cp -s by cp -P...

I do not think they are the same: From [GNU manpage](https://linux.die.net/man/1/cp):

> -s   make symbolic links instead of copying
> -P   never follow symbolic links in SOURCE
> -L   always follow symbolic links in SOURCE

So here is an example using all of them (output at the end is simplified):

```bash
$ touch original
$ ln -s original symlink
$ cp -P symlink P
$ cp -L symlink L
$ cp -s symlink s
$ ls -l
original
symlink -> original
P -> original
L
s -> symlink
```

So `-P` copies the symlink, `-L` copies the linked file, and `-s` creates a symlink to the symlink.

**Comment by der-john** [comment]
was fixed by #2169.
